task8.py

```diff
@@ -0,0 +1,17 @@
+'''
+You are given a list of numbers and your task is to sort the list without any inbuilt method of python.
+ Input:
+numbers = [2, 5, 6, 1, 3, 8, 9, 10]
+ Output:
+[1, 2, 3, 5, 6, 8, 9, 10]
+'''
+
+#sort list without in-built function
+numbers = [2, 5, 6, 1, 3, 8, 9, 10]
+for i in range (len(numbers)):
+    for j in range(len(numbers)):
+        if(numbers[i] < numbers[j]):
+            temp = numbers[i]
+            numbers[i] = numbers[j]
+            numbers[j] = temp
```

Comment:
Would you please tell me which algorithm you have used here?

Reply:
Swapping using third variable

Reply:
It's a logic and description. I asked sorting algorithm name. Which sorting algorithm you are using here to sort the list?

Reply:
bubble sort algorithm